### PR TITLE
Pull down coredistool package on Windows

### PR DESCRIPTION
**Pull request by sejongoh**

### CODE REVIEW

File: tests/runtest.cmd [hunk 1]
```diff
@@ -177,6 +177,16 @@ xcopy /s "%__BinDir%" "%CORE_ROOT%"
 
 :SkipCoreRootSetup
 
+:: Pull down dependent packages needed for testing
+setlocal
+if defined __TestEnv call %__TestEnv%
+if defined COMPlus_GCStress set __Result=true
+endlocal & set __IsGCTest=%__Result%
```
Comment (kyulee1) [inline tests/runtest.cmd]: I'm just curious in what situation we expect ```__TestEnv`` and why not just checking ```COMPlus_GCStresss``` like Unix case.

File: tests/runtest.cmd [hunk 2]
```diff
@@ -177,6 +177,16 @@ xcopy /s "%__BinDir%" "%CORE_ROOT%"
...
+if "%__IsGCTest%"=="true" (
+    call tests\setup-runtime-dependencies.cmd /outputdir %CORE_ROOT%
```
Comment (kyulee1) [inline tests/runtest.cmd]: You created ```setup-gcstress.sh``` for Unix before. Why not ```setup-gcstress.cmd``` for Windows?

File: tests/setup-runtime-dependencies.cmd [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    "dependencies": { ^
+    "Microsoft.NETCore.CoreDisTools": "1.0.0-prerelease-00001" ^
+    }, ^
+    "frameworks": { "dnxcore50": { } } ^
+    } > "%__JasonFilePath%"
+
+REM Download the package
+echo Downloading CoreDisTools package
+echo on
+call "%__DotNetCmd%" restore "%__JasonFilePath%" --source https://dotnet.myget.org/F/dotnet-core/ --packages "%__PackageDir%"
```
Comment (kyulee1) [inline tests/setup-runtime-dependencies.cmd]: It appears now "--runtime" is obsolete for CLI which you used in Unix script ```setup-gcstress.sh```. Can you update that file as well that works with the up-to-date CLI tool?

File: tests/setup-runtime-dependencies.cmd [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+REM =========================================================================================
+
+:Usage
+echo.
+echo Download coredistool for GC stress testing
+echo.
+echo Usage:
+echo     %__ThisScriptShort% /outputdir ^<coredistools_lib_install_path^>
+echo.
+exit /b 1
```
Comment (kyulee1) [inline tests/setup-runtime-dependencies.cmd]: Nit: Next line.

File: tests/runtest.cmd (hunk 1 above)
Comment (sejongoh) [inline tests/runtest.cmd]: runtest.cmd uses __TestEnv to apply environment variables only to tests, not affecting build tools and test harness.

File: tests/setup-runtime-dependencies.cmd (hunk 3 above)
Comment (sejongoh) [inline tests/setup-runtime-dependencies.cmd]: Yeah, ''--runtime' is not required now. I will update the Linux script.

File: tests/runtest.cmd (hunk 2 above)
Comment (sejongoh) [inline tests/runtest.cmd]: We just picked a general name to add any future runtime dependencies here. I will use the same name for both the scripts.

**Comment by sejongoh** [comment]
@kyulee1 PTAL

**Comment by kyulee1** [comment]
I don't see any change in .sh file.

**Comment by sejongoh** [comment]
I added the sh file. I was thinking of a separate PR for Linux changes but it looks good to have all the changes here.

**Comment by kyulee1** [comment]
LGTM

**Pull request merged**